commerce-frontend-js/src/main/resources/META-INF/resources/js/smart_table/SmartTable.es.js

```diff
@@ -37,9 +35,10 @@ function WrappingCard(props) {
 // 	return endpoint;
 // }
 
-function Wrapper(props) {
+function SmartTable(props) {
```

Comment:
Let's find a better name for this component instead of the SmartTable. This component is a Wrapper on top of ManagmentBar, Table, PaginationBar. Suggestions:  ManagedTable , FilterableTable, DataSetDisplay.
Maybe DataSetDisplay is better thinking in the future evolution that will contain other display styles ex list, card and so on.  @ethanbustad Ideas?